### PR TITLE
v2.5.0.0: "Hopefully fixing the mess" edition (BREAKING CHANGES!)

### PR DESCRIPTION
**Pull request by pablouser1**
**If you are a webmaster**: This version includes **BREAKING CHANGES**, please check the pull request below for details on all the changes and the `docker-compose.yml` file for the service changes.
Also, check the `Self hosting` wiki page for instruction on how to stop getting captchas.

Hi! I've been working on a new version for the [scraper](https://github.com/pablouser1/TikScraperPHP) that (should) fix ProxiTok.

Please check the pull request from the other repo (pablouser1/TikScraperPHP#19) for the entire changelog.
This version of ProxiTok only changes the necessary to make ProxiTok work.

### CODE REVIEW

**Comment by wedcf120** [comment]
https://proxitok.pussthecat.org/@enola.bedard
https://proxitok.pussthecat.org/@enola.bedard/video/7405356541940894982

https://proxitok.pussthecat.org/@ladbible
https://proxitok.pussthecat.org/@ladbible/video/7405139478270020896
 
video page seems worked. but profile page cant be grab?

**Comment by pablouser1** [comment]
That instance is still using the old version of ProxiTok and TikScraper, please check the about page and see the versions of both the frontend (should be >= v2.5.0.0) and the scraper (should be >= 2.6.0.0)

**Comment by strangersdangers** [comment]
Why were the trending and discover buttons removed? Also, I keep getting 500 errors when trying to view a video or a profile, other instances that don't have this new version did not experience this.

File: templates/components/themes/card.latte [hunk 1]
```diff
@@ -67,4 +67,4 @@
     </footer>
   </div>
 </div>
-<script src="{static('js', 'themes/card.js')}"></script>
```
Comment (reynaldamabler5-prog) [inline templates/components/themes/card.latte]: -<script  src="{static( ' js' ,  'themes/card. js')}"></script>

**Pull request merged**